code/game/objects/items/devices/techno_tribalism.dm

```diff
@@ -0,0 +1,86 @@
+/obj/item/device/techno_tribalism
+	name = "Techno-Tribalism Enforcer"
+	desc = "Feeds on rare tools, tool mods and other tech items, according to their loot rarity and tag. When is feeded enough, will produce an oddity."
+	icon = 'icons/obj/faction_item.dmi'
+	icon_state = "techno_tribalism"
+	item_state = "techno_tribalism"
+	origin_tech = list(TECH_MATERIAL = 10, TECH_ENGINEERING = 7, TECH_POWER = 2)
+	price_tag = 20000
+	var/list/oddity_stats = list(STAT_MEC = 0,STAT_COG = 0,STAT_BIO = 0,STAT_ROB = 0,STAT_TGH = 0,STAT_VIG = 0)
+	var/last_produce = -30 MINUTES
+	var/items_count = 0
+	var/cooldown = 30 MINUTES
+
+/obj/item/device/techno_tribalism/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(items_count < 5)
+		if(istype(W, /obj/item/weapon/tool))
+
+			if(istype(W, /obj/item/weapon/tool/sword) || istype(W, /obj/item/weapon/tool/knife) || istype(W, /obj/item/weapon/tool/hammer))
+				var/high_force = max(W.force, W.force_unwielded, W.force_wielded)
+				oddity_stats[STAT_ROB] += round(high_force / 5)
```

Comment:
This can get out of control really quickly, you need to provide some sort of a cap per item.